docker-compose.yml

```diff
@@ -53,6 +53,7 @@ services:
     volumes:
       - .:/tileset
       - ./data:/import
+      - ./data:/export
```

Comment:
Just curious, what does this do?

Reply:
This also came from the original MapTiler repo https://github.com/openmaptiles/openmaptiles/pull/1071